frontend/test/pytest/test_template.py

```diff
@@ -168,6 +199,23 @@ def basic_entangler_layers(weights):
     params = jnp.array([[jnp.pi, jnp.pi, jnp.pi]])
     interpreted_fn = qml.QNode(basic_entangler_layers, device)
     jitted_fn = qjit(interpreted_fn)
+
+    assert np.allclose(interpreted_fn(params), jitted_fn(params))
+
+
+@pytest.mark.filterwarnings("ignore::pennylane.PennyLaneDeprecationWarning")
```

Comment:
If this is deprecated, why we need to test it at all?

Reply:
I did not add this test, so I'm leaving it as it is.

Reply:
It seems this is not on master any more

Reply:
Oh good catch. I think this warning was supposed to be removed in a recent merge conflict that removed a bunch of old tests.